README.md

```diff
@@ -339,6 +339,14 @@ Customize the output image/video directory location:
 $ git-sim --media-dir=path/to/output status
 ```
 
+Optionally, set the environment variable `git_sim_media_dir` to default that as the media directory, if no `--media-dir` is provided.
```

Comment:
Let's update to something like:

Optionally, set the environment variable `git_sim_media_dir` to set a global default media directory, to be used if no `--media-dir` is provided. Simulated output images/videos will be placed in this location, in subfolders named with the corresponding repo's name.